packages/dataviews/src/stories/index.story.js

```diff
@@ -49,6 +49,20 @@ const fields = [
 		maxWidth: 400,
 		enableHiding: false,
 	},
+	{
+		header: 'Type',
+		id: 'type',
+		getValue: ( { item } ) => item.type,
+		maxWidth: 400,
+		enableHiding: false,
+		type: 'enumeration',
+		elements: [
```

Comment:
The DataViews component can't know if the filtering is happening client side or server side, which means it can't know if the available values are exhaustive. I'm fine with this being defined on the field personally.

Reply:
I pushed the filtering for now.